snuba/cli/subscriptions_scheduler.py

```diff
@@ -48,6 +48,11 @@
 @click.option("--schedule-ttl", type=int, default=60 * 5)
 @click.option("--log-level", help="Logging level to use.")
 @click.option("--delay-seconds", type=int)
+@click.option(
+    "--stale-threshold-seconds",
+    type=int,
+    help="Skip scheduling if timestamp is beyond this threshold compared to the system time",
+)
```

Comment:
Should we add this to the subscriptions executor as well? It is possible that the scheduler is running ok but the executor was down for 5 minutes. Shouldn't the executor not run the queries in that case?

Reply:
Yes, I will add the same to the executor too